src/createStore.js

```diff
@@ -73,7 +73,10 @@ export default function createStore(reducer, initialState, enhancer) {
   function getState() {
     return currentState
   }
-
+  
+  if (typeof getState !== 'function' || getState == 'undefined') {
```

Comment:
`getState` is a locally defined function. It is guaranteed to be a function by the code unless I’m missing something.